packages/config/config/devices/0x0190/adc-t_3000.json

```diff
@@ -0,0 +1,801 @@
+			"minValue": 0,
+			"maxValue": 95,
+			"defaultValue": 30
+		},
+		{
+			"#": "12",
+			"label": "Fan Circulation Period",
+			"valueSize": 2,
+			"unit": "minutes",
+			"minValue": 10,
+			"maxValue": 1440,
+			"defaultValue": 60,
+			"unsigned": true
+		},
+		{
+			"#": "13",
+			"label": "Fan Circulation Duty Cycle",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 25
+		},
+		{
+			"#": "14",
+			"label": "Fan Purge Time",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 3600,
+			"defaultValue": 60
+		},
+		{
+			"#": "15",
+			"label": "Maximum Heat Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 35,
+			"maxValue": 95,
+			"defaultValue": 95
+		},
+		{
+			"#": "16",
+			"label": "Minimum Heat Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 35,
+			"maxValue": 95,
+			"defaultValue": 35
+		},
+		{
+			"#": "17",
+			"label": "Maximum Cool Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 50,
+			"maxValue": 95,
+			"defaultValue": 95
+		},
+		{
+			"#": "18",
+			"label": "Minimum Cool Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 50,
+			"maxValue": 95,
+			"defaultValue": 50
+		},
+		{
+			"#": "19",
+			"label": "Thermostat Lock",
+			"description": "Lock out physical thermostat controls.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Full Lock",
+					"value": 1
+				},
+				{
+					"label": "Partial Lock",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "20",
+			"label": "Compressor Delay",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 60,
+			"defaultValue": 5
+		},
+		{
+			"#": "23",
+			"label": "Temperature Display Units",
+			"description": "Celsius or Farenheit for temperature display.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Farenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "24",
+			"label": "HVAC Modes Enabled",
+			"description": "Which heating/cooling modes are available.",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 31,
+			"defaultValue": 15,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off, Heat",
+					"value": 3
+				},
+				{
+					"label": "Off, Cool",
+					"value": 5
+				},
+				{
+					"label": "Off, Heat, Cool",
+					"value": 7
+				},
+				{
+					"label": "Off, Heat, Cool, Auto",
+					"value": 15
+				},
+				{
+					"label": "Off, Heat, Emergency Heat",
+					"value": 19
+				},
+				{
+					"label": "Off, Heat, Cool, Emergency Heat",
+					"value": 23
+				},
+				{
+					"label": "Off, Heat, Cool, Auto, Emergency Heat",
+					"value": 31
+				}
```

Comment:
Are these the only possible combinations or should this be split into partial parameters instead for free combination of each possibility?

Reply:
Only these combinations are possible per the manual